cli/cmd/root.go

```diff
@@ -30,7 +30,7 @@ func Execute() error {
 func NewRootCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:              "constellation",
-		Short:            "Manage your Constellation cluster",
+		Short:            "manage your Constellation cluster",
```

Comment:
Pls revert and apply style from https://github.com/spf13/cobra/blob/main/user_guide.md#using-the-cobra-library

Reply:
You can also see the style used by the predefined commands `help` and `completion`. We need to stick to the same style to be consistent.

Reply:
my bad regarding the Cobra document, I thought it was consistent ;)